subgraphs/gtdb.yaml

```diff
@@ -0,0 +1,7 @@
+id: GTDB
+name: Genome Taxonomy Database
+category: taxonomy
+url: https://gtdb.ecogenomic.org/
+vertices: [gtdb_organism, gtdb_taxon]
```

Comment:
Splitting this into 2 collections seems odd at first glance

Reply:
Removed gtdb_organism